dockerhub-readme.md

```diff
@@ -0,0 +1,141 @@
+
+### Container Ports
+
+`rapidsai/notebooks` exposes port `8888` for the [JupyterLab notebook server](https://jupyter.org/).
+
+### Environment Variables
+
+The following environment variables can be passed to the `docker run` commands:
+
+- `EXTRA_CONDA_PACKAGES` - used to install additional `conda` packages in the container. Use a space separated list of values
+- `CONDA_TIMEOUT` - how long (in seconds) the `conda` command should wait before exiting
+- `EXTRA_PIP_PACKAGES` - used to install additional `pip` packages in the container. Use a space separated list of values
+- `PIP_TIMEOUT` - how long (in seconds) the `pip` command should wait before exiting
+
+Example:
+
+```sh
+$ docker run \
+    --rm \
+    -it \
+    --gpus all \
+    -shm-size=1g --ulimit memlock=-1 --ulimit stack=67108864 \
+    -e EXTRA_CONDA_PACKAGES="jq" \
+    -e EXTRA_PIP_PACKAGES="beautifulsoup4" \
+    -p 8888:8888 \
+    rapidsai/notebooks:23.08-cuda11.8-py3.10
+```
+
+### Bind Mounts
+
+Mounting files/folders to the locations specified below provide additional functionality for the images.
+
+- `/home/rapids/environment.yml` - a YAML file that contains a list of dependencies that will be installed by `conda`. The file should look like:
+
+```yml
+dependencies:
+  - beautifulsoup4
+  - jq
+```
+
+Example:
+
+```sh
+$ docker run \
+    --rm \
+    -it \
+    --gpus all \
+    -shm-size=1g --ulimit memlock=-1 --ulimit stack=67108864 \
+    -v $(pwd)/environment.yml:/home/rapids/environment.yml \
+    rapidsai/base:23.08-cuda11.8-py3.10
+```
+
+### Use JupyterLab to Explore the Notebooks
+
+The `rapidsai/notebooks` container has notebooks for the RAPIDS libraries in `/home/rapids/notebooks`.
+
+### Extending RAPIDS Images
+
+All RAPIDS images use `conda` as their package manager, and all RAPIDS packages are available in the `base` conda environment. These image run as the `rapids` user.
+
+### Access Documentation within Notebooks
+
+You can check the documentation for RAPIDS APIs inside the JupyterLab notebook using a `?` command, like this:
+```
+[1] ?cudf.read_csv
+```
+This prints the function signature and its usage documentation. If this is not enough, you can see the full code for the function using `??`:
+```
+[1] ??pygdf.read_csv
+```
+Check out the RAPIDS [documentation](https://docs.rapids.ai/) for more detailed information.
+
+## More Information
+
+Check out the [RAPIDS User Guides](https://docs.rapids.ai/user-guide) and [XGBoost](https://xgboost.readthedocs.io/en/latest/) API docs.
+
+## Where can I get help or file bugs/requests?
+
+Please submit issues with the container to this GitHub repository: [https://github.com/rapidsai/docker](https://github.com/rapidsai/docker/issues/new)
+
+For issues with RAPIDS libraries like cuDF, cuML, RMM, or others file an issue in the related GitHub project.
+
+Additional help can be found on [Stack Overflow](https://stackoverflow.com/tags/rapids) or [Google Groups](https://groups.google.com/forum/#!forum/rapidsai).
```

Comment:
I didn't even know we have a Google Group/clicking on the link gets me a `content unavailable` flag. I'd remove for now, and if/when we spawn discussions we can add it here.

```suggestion
Additional help can be found on [Stack Overflow](https://stackoverflow.com/tags/rapids).
```